etc/emqx.conf

```diff
@@ -328,91 +328,61 @@ rpc.socket_keepalive_count = 9
 
-## Sets the log dir.
-##
-## Value: Folder
-log.dir = {{ platform_log_dir }}
-
-## Where to emit the console logs.
+## Where to emit the logs.
+## Enable the console (standard output) logs.
 ##
 ## Value: off | file | console | both
-## - off: disabled
-## - file: write to file
-## - console: write to stdout
-## - both: file and stdout
-log.console = console
+## - off: disable logs entirely
+## - file: write logs to file
+## - console: write logs to standard I/O
+## - both: write logs both to file and standard I/O
+log.to = console
 
-## Sets the severity level of console log.
+## The log severity level.
 ##
 ## Value: debug | info | notice | warning | error | critical | alert | emergency
 ##
-## Default: error
-log.console.level = error
-
-## The file where console logs will be writed to, when 'log.console' is set as 'file'.
+## Note: Only the messages with severity level greater than or equal to
+##       this level will be logged.
 ##
-## Value: File Name
-## log.console.file = {{ platform_log_dir }}/console.log
-
-## Maximum file size for console log.
-##
-## Value: Number(bytes)
-## log.console.size = 10485760
-
-## The rotation count for console log.
-##
-## Value: Number
-## log.console.count = 5
+## Default: error
+log.level = error
 
-## The file where info logs will be writed to.
+## The dir for log files.
 ##
-## Value: File Name
-## log.info.file = {{ platform_log_dir }}/info.log
+## Value: Folder
+log.dir = {{ platform_log_dir }}
 
-## Maximum file size for info log.
+## The log filename for logs of level specified in "log.level".
 ##
-## Value: Number(bytes)
-## log.info.size = 10485760
+## Value: String
+## Default: emqx.log
+log.file = emqx.log
 
-## The rotation count for info log.
+## Maximum size of each log file.
 ##
 ## Value: Number
-## log.info.count = 5
-
-## The file where error logs will be writed to.
-##
-## Value: File Name
-log.error.file = {{ platform_log_dir }}/error.log
-
-## Maximum file size for error log.
-##
-## Value: Number(bytes)
-log.error.size = 10485760
+## Default: 10M
+## Supported Unit: KB | MB | G
+log.rotation.size = 10MB
 
-## The rotation count for error log.
+## Maximum rotation count of log files.
 ##
 ## Value: Number
-log.error.count = 5
+## Default: 5
+log.rotation.count = 5
 
-## Enable the crash log.
-##
-## Value: on | off
-log.crash = on
-
-## The file for crash log.
+## To create additional log files for specific log levels.
 ##
 ## Value: File Name
-log.crash.file = {{ platform_log_dir }}/crash.log
-
-## Enable syslog.
-##
-## Values: on | off
-log.syslog = on
-
-## Sets the severity level for syslog.
-##
-## Value: debug | info | notice | warning | error | critical | alert | emergency
-log.syslog.level = error
+## Format: log.$level.file = $filename,
+##         where "$level" can be one of: debug, info, notice, warning,
+##                                       error, critical, alert, emergency
+## Note: Log files for a specific log level will contain all the logs
```

Comment:
I'm not quite sure why we need this extra log config if we already have `log.file`.
Especially when the severity level filter logic (higher than or equal to) is exactly the same.